dnf/cli/commands/repoquery.py

```diff
@@ -657,6 +657,8 @@ def __init__(self, pkg):
 
     def __getattr__(self, attr):
         atr = getattr(self._pkg, attr)
+        if atr is None:
```

Comment:
Please move this change into another commit. It is change of output independent on %{reason} feature.

Reply:
Done